Sources/Jetworking/Client/Client.swift

```diff
@@ -94,9 +94,17 @@ public final class Client {
 
     // MARK: - Methods
     @discardableResult
-    public func get<ResponseType: Decodable>(endpoint: Endpoint<ResponseType>, _ completion: @escaping RequestCompletion<ResponseType>) -> CancellableRequest? {
+    public func get<ResponseType: Decodable>(
+        endpoint: Endpoint<ResponseType>,
+        additionalHeaderFields: [String: String] = [:],
+        _ completion: @escaping RequestCompletion<ResponseType>
+    ) -> CancellableRequest? {
         do {
-            let request: URLRequest = try createRequest(forHttpMethod: .GET, and: endpoint)
+            let request: URLRequest = try createRequest(
+                forHttpMethod: .GET,
+                and: endpoint,
+                additionalHeaderFields: additionalHeaderFields
```

Comment:
Should this also be named `and` instead of `additionalHeaderFields`?

Reply:
Since the headers are a "untyped" Dictionary the only possibility would be `andAdditionalHeaderFields`. What do you think?

Reply:
Then please use `andAdditionalHeaderFields ` to be more precise.

Reply:
Ok 👌